### ISSUE TITLE
org.openqa.selenium.WebDriverException: cannot create session chrome-80.0 on any hosts after 2 attempt(s) 

### ISSUE THREAD

**Issue by ankur1233** [aerokube/ggr#307]
I'm trying to create a selenium cluster using ggr.

It's throwing below error:
**org.openqa.selenium.WebDriverException: cannot create session chrome-80.0 on any hosts after 2 attempt(s)** 

**Will get same error when I set browse name = firefox& version 74.0: can not create session firefox-74.0**

Test code:- 
(DesiredCapabilities capabilities = new DesiredCapabilities();
		capabilities.setBrowserName("chrome");
		capabilities.setVersion("80.0");
		capabilities.setCapability("enableVNC", true);
		capabilities.setCapability("enableVideo", false);

	    RemoteWebDriver driver = new RemoteWebDriver(
		    URI.create("http://test:test-password@15.206.128.235:4444/wd/hub").toURL(), 
		    capabilities
		);)
    	
ggr instance :- 
![image](https://user-images.githubusercontent.com/18676332/77817042-e3d26d00-70ed-11ea-9b77-7d9eaa628389.png)

HUB -1 instance 
![image](https://user-images.githubusercontent.com/18676332/77817079-1aa88300-70ee-11ea-834c-5526b3b8fff6.png)]

HUB-2 instance 
![image](https://user-images.githubusercontent.com/18676332/77817099-37dd5180-70ee-11ea-98e5-c904e0f4a3d6.png)

test.xml file 
![image](https://user-images.githubusercontent.com/18676332/77817133-770ba280-70ee-11ea-8400-3bf32f5f53a5.png)

I have created password using this command:- **$ htpasswd -bc /etc/grid-router/users.htpasswd test test-password**

Please let me know what I'm doing wrong. 

When I try directly to connect to selenoid hub. it works fine 

(Test code ) --> (HUB-1)Selenoid hub ---> Works

(Test code)-- ggr(HUB-1,HUB-2) ---->(HUB-1/HUB-2) ---> not works

**Comment by ankur1233** [comment]
@vania-pooh @vania-pooh

**Comment by vania-pooh** [comment]
@ankur1233 take a look at Selenoid logs to see whether your requests arrive there from Ggr.

**Comment by ankur1233** [comment]
> @ankur1233 take a look at Selenoid logs to see whether your requests arrive there from Ggr.

I **don't see any logs** in the selenoid HUB-3 ec2 Instance console when **I run through ggr.**

ggr ec2 instance logs

![image](https://user-images.githubusercontent.com/18676332/77825980-a9d78a00-7132-11ea-91df-1649c7f59a62.png)



But when I **directly** connect to HUB-3 ec2 instance URL to my code like this.
	
DesiredCapabilities capabilities = new DesiredCapabilities();
		capabilities.setBrowserName("chrome");
		capabilities.setVersion("80.0");
		capabilities.setCapability("enableVNC", true);
		capabilities.setCapability("enableVideo", false);
RemoteWebDriver driver = new RemoteWebDriver(
			    URI.create("http://13.233.81.164:4444/wd/hub").toURL(), 
			    capabilities
			);)

It logs below msg in HUB HUB-2 ec2 instance console.
![image](https://user-images.githubusercontent.com/18676332/77825928-58c79600-7132-11ea-891c-85a167a9ff6c.png)

**Comment by vania-pooh** [comment]
@ankur1233 anyway ggr failed to create session on both hosts. Please check that you can create it manually on hosts where Ggr failed to do that.

**Comment by ankur1233** [comment]
Yes manually I can create host & run as well you can see my screenshot attach for manually run selenoid in host 

refer below image given this comment: **it logs below msg in HUB HUB-2 ec2 instance console.**

**Comment by ankur1233** [comment]
![image](https://user-images.githubusercontent.com/18676332/77844080-f9649700-71c0-11ea-829f-1ea6dcc79b72.png)

**Comment by ankur1233** [comment]
@vania-pooh

**Comment by vania-pooh** [comment]
@ankur1233 this is a wrong log because you should have user `test` in Selenoid log but you have `unknown`. However so far as on every host Ggr spends 30 seconds I would suggest that for some reason Selenoid is not creating session.

**Comment by ankur1233** [comment]
@vania-pooh  **I would suggest that for some reason Selenoid is not creating session.** 

Can you please help me out. How can I debug this issue.

Please let me know what I'm doing wrong so I can get you the exact logs that you are expecting.

**Comment by vania-pooh** [comment]
@ankur1233 try to find logs starting with something like:
```
[x] [NEW_REQUEST_ACCEPTED] [test] [x.x.x.x]
```

**Comment by ankur1233** [comment]
Hey @vania-pooh 

if have changed my test.xml file which contains now only **2 host server and 1 region**  previously it was large config with 4 regions and 2 servers in every region :

![image](https://user-images.githubusercontent.com/18676332/78183109-00bbc700-7485-11ea-8306-0d97e73bdc40.png)

In that, I was able to see logs which you were expecting 

![image](https://user-images.githubusercontent.com/18676332/78183334-5b552300-7485-11ea-82c8-c4ab55a0b807.png)

But still, I'm getting error in my eclipse console.

org.openqa.selenium.**WebDriverException: cannot create session firefox-74.0 on any hosts after 2** attempt(s) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 60.13 seconds
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'LAPTOP-B5FT9H4U', ip: '192.168.99.1', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_181'
Driver info: driver.version: RemoteWebDriver

**Comment by ankur1233** [comment]
@vania-pooh
